src/components/portfolio/DetailsHeader.tsx

```diff
@@ -80,7 +81,9 @@ const DetailsHeader: React.FC<DetailsHeaderProps> = ({
             <ArrowBackIosIcon fontSize="large" />
           </IconButton>
         </div>
-        <div>{name && <span className={classes.title}>{name}</span>}</div>
+        <div className={classes.title}>
+          {name && <LimitedString value={name} />}
+        </div>
```

Comment:
I think we should allow names to be longer in the details header
The only other thing is that you can't hover the apex legends, but I'm working on that right now.
Other than that, looks good.

```suggestion
        <div className={classes.title}>
          {name && <LimitedString value={name} length={60}/>}
        </div>
```